src/main/java/io/apimatic/core/utilities/ConversionHelper.java

```diff
@@ -0,0 +1,212 @@
+ * SDK.
+ */
+public final class ConversionHelper {
+
+    /**
+     * Private constructor to prevent instantiation of this utility class.
+     */
+    private ConversionHelper() {
+        // Prevent instantiation
+    }
+
+    /**
+     * Converts a single object to the specified type.
+     *
+     * @param <S>                the type to convert to.
+     * @param value              the object to convert.
+     * @param conversionFunction the function to apply for conversion.
+     * @return the converted object of type {@code S}, or null if conversion fails.
+     */
+    public static <S> S convertToSimpleType(Object value, Function<Object, S> conversionFunction) {
+        try {
+            return conversionFunction.apply(value);
+        } catch (Exception e) {
+            return null;
+        }
+    }
+
+    /**
+     * Converts a map of objects to a map of the specified type.
+     *
+     * @param <S>                the type of values in the resulting map.
+     * @param value              the map of objects to convert.
+     * @param conversionFunction the function to apply for conversion of each value.
+     * @return a map with values converted to type {@code S}, or null if conversion
+     *         fails.
+     */
+    @SuppressWarnings("unchecked")
+    public static <S> Map<String, S> convertToMap(Object value,
+            Function<Object, S> conversionFunction) {
+        if (value == null) {
+            return null;
+        }
+
+        try {
+            Map<String, Object> valueMap = (Map<String, Object>) value;
+            return valueMap.entrySet().stream()
+                    .map(entry -> new AbstractMap.SimpleEntry<>(entry.getKey(),
+                            convertToSimpleType(entry.getValue(), conversionFunction)))
+                    .filter(entry -> entry.getValue() != null)
+                    .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+        } catch (Exception e) {
+            return null;
+        }
+    }
+
+    /**
+     * Converts a list of objects to a list of the specified type.
+     *
+     * @param <S>                the type of elements in the resulting list.
+     * @param value              the list of objects to convert.
+     * @param conversionFunction the function to apply for conversion of each item.
+     * @return a list with elements converted to type {@code S}, or null if
+     *         conversion fails.
+     */
+    @SuppressWarnings("unchecked")
+    public static <S> List<S> convertToArray(Object value,
+            Function<Object, S> conversionFunction) {
+        if (value == null) {
+            return null;
+        }
+
+        try {
+            List<Object> valueList = (List<Object>) value;
+            return valueList.stream().map(item -> convertToSimpleType(item, conversionFunction))
+                    .filter(item -> item != null).collect(Collectors.toList());
```

Comment:
Intelij is suggesting using the following: 

`.filter(Objects::nonNull).collect(Collectors.toList());`

Reply:
Added your suggestion in the Core library